docs/wiki/Native-BTC-Wallet-Recovery-and-Rescanning.md

```diff
@@ -1,38 +1,64 @@
+In this case, the Wallets page (`/wallets`) will still show the wallet since it was previously configured, just that it is not synchronized and cannot be unlocked or connected.  It will look like this:
 
-![wallets view](https://user-images.githubusercontent.com/9373513/149597739-06318c29-f6be-4837-bb6f-2055971be8cf.png)
+<img alt="BTC Wallet View" src="./images/unloaded-btc-wallet.png" width="300"/>
 
-This may be solvable by simply clicking the gears to open the settings box, and then click Submit:
+To reinitialize the wallet, [open the wallet settings](#opening-the-btc-wallet-settings) and click the `Recover` button:
 
-![wallet settings](https://user-images.githubusercontent.com/9373513/149597817-abbcbf5f-d05f-4b48-996f-485030d2a74c.png)
+<img alt="BTC Wallet Settings View" src="./images/btc-wallet-settings.png" width="300"/>
 
-The dexc.log should say:
+The `dexc.log` should say:
 
-> [INF] CORE: Initializing a built-in btc wallet
+> [DBG] CORE[btc][SPV]: Starting native BTC wallet...
 
 The wallet will begin re-synchronizing.  This may take several minutes.
 
-If this process results in errors, it may be necessary to go to the folder containing the wallet data and **delete the wallet.db so it may be regenerated** from your application seed.  First SHUTDOWN DEX. Next, for mainnet, on Linux, this would be in the folder `~/.dexc/mainnet/assetdb/btc/mainnet/`.  There may be a wallet.db file in this folder, which you should rename wallet.db.bak.  Now startup and login to DEX.  Go to the Wallets page to see wallet status, which should re-synchronize.  Check notifications and logs for any errors.
+**IF** this process results in errors, it may be necessary to go to the folder containing the wallet data and **delete the wallet.db so it may be regenerated** from your application seed.  To do this:
+
+1. Shutdown DEX.
+2. Open the wallet directory. For mainnet, this would be:
+    - `~/.dexc/mainnet/assetdb/btc/mainnet/` on Linux
+    - `~/Library/Application\ Support/Dexc/mainnet/assetdb/btc/mainnet/` on MacOS 
+    - `C:\Users\<Username>\AppData\Dexc\mainnet\assetdb\btc\mainnet` on Windows.
+
+   There may be a wallet.db file in this folder, which you should rename wallet.db.bak.
+
+3. Startup and login to DEX.
+4. [Open the BTC wallet settings](#opening-the-btc-wallet-settings) and click `Recover` again.
+
+If you still encounter any error in the logs or notification, you can reach out to us on our chat channels. We will be happy to help you out.
 
 ## Rescanning the wallet
 
-There will be a Rescan button added to the Wallets page, but for now, you can force the wallet to rescan in one of two ways.  First, SHUTDOWN DEX.  Then do one of the following:
+If you have sent funds to your wallet, but they are not showing up in DEX, it may be necessary to rescan the wallet.  This is a process that can take several hours, depending on the number of transactions in the wallet.
```

Comment:
~~`If you have sent funds to your wallet, but they are not showing up in DEX, it may be necessary to rescan the wallet.`~~

Does this happen?

Maybe just say something like...

`In rare circumstances, the wallet may need to rescan the blockchain for missed transactions.`